src/unittest/extproc_test.cc

```diff
@@ -263,9 +263,9 @@ class segfault_job_t : public base_crash_job_t {
 
 private:
     static bool worker_fn(read_stream_t *, write_stream_t *) {
-        int bad_data = *reinterpret_cast<volatile int*>(NULL);
```

Comment:
You can also try compiling with the `ALLOW_WARNINGS=1` option to `gmake`. Our default settings are quite strict when it comes to compiler warnings, and we usually treat them as errors.